CMakeLists.txt

```diff
@@ -33,8 +33,11 @@ set (LOOM_INCLUDE_FOLDERS
     ${CMAKE_SOURCE_DIR}/loom/vendor/sqlite3
     ${CMAKE_SOURCE_DIR}/loom/vendor/facebook/ios/FacebookSDK.framework/Headers
     ${CMAKE_SOURCE_DIR}/loom/vendor/parse/ios/Parse.framework/Headers
+    ${CMAKE_SOURCE_DIR}/loom/vendor/dxsdk/Include
     )
 
+include_directories(${LOOM_INCLUDE_FOLDERS})
```

Comment:
If I remove any or both of those lines I receive this error:
```"D:\Development\LoomSDK\cmake_msvc\LoomEngine.sln" (default target) (1) ->
"D:\Development\LoomSDK\cmake_msvc\ALL_BUILD.vcxproj.metaproj" (default target) (2) ->
"D:\Development\LoomSDK\cmake_msvc\loom\vendor\sdl2\SDL2-static.vcxproj" (default target) (6) ->
(ClCompile target) ->
  D:\Development\LoomSDK\loom\vendor\sdl2\src\audio\xaudio2\SDL_xaudio2.c(68): fatal error C1083: Cannot open i
nclude file: 'dxsdkver.h': No such file or directory [D:\Development\LoomSDK\cmake_msvc\loom\vendor\sdl2\SDL2-s
tatic.vcxproj]```
So it looks like they have to stay, at least with the current solution of detecting DirectX.